stats/stats/src/charts/chart.rs

```diff
@@ -42,43 +45,35 @@ pub trait Chart: Sync {
     fn relevant_or_zero(&self) -> bool {
         false
     }
-    fn drop_last_point(&self) -> bool {
-        self.chart_type() == ChartType::Line
+    /// Number of last values that are considered approximate.
+    /// (ordered by time)
+    ///
+    /// E.g. how many end values should be recalculated on (kinda)
+    /// lazy update (where `get_last_row` is retrieved successfully).
+    /// Also controls marking points as approximate when returning data.
+    ///
+    /// ## Value
+    ///
+    /// Usually set to 1 for line charts. Currently they have resolution of
+    /// 1 day. Also, data for portion of the (last) day has to be recalculated
+    /// on the next day.
+    ///
+    /// I.e. for number of blocks per day, stats for current day (0) are
+    /// not complete because blocks will be produced till the end of the day.
+    ///    |===|=  |
+    /// day -1   0
+    fn approximate_until_updated(&self) -> u64 {
```

Comment:
I didnt get why name of this variable `approximate_until_updated`. maybe something like `approximate_last_points` or `approximate_last_values_count` of `approximate_last_points_number`

Reply:
I didn't think about trimming future points while naming it; good catch ty